utilities/src/main/java/io/syndesis/qe/utils/mqtt/MqttUtils.java

```diff
@@ -0,0 +1,61 @@
+package io.syndesis.qe.utils.mqtt;
+
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+
+import java.util.Optional;
+
+@Slf4j
+public class MqttUtils {
+    private static final String broker = "tcp://localhost:1883";
+
+
+    public MqttClient createReceiver(String clientName, String topic) throws MqttException {
+        final String clientId = clientName;
+        MqttClient sampleClient = new MqttClient(broker, clientId, new MemoryPersistence());
+        MqttConnectOptions connOpts = new MqttConnectOptions();
+        connOpts.setCleanSession(false);
+
+        Optional<Account> optional = AccountsDirectory.getInstance().getAccount("QE MQTT");
+        if (optional.isPresent()) {
+            log.info("Setting username and password for QE MQTT client");
+            connOpts.setUserName(optional.get().getProperties().get("userName"));
+            connOpts.setPassword(optional.get().getProperties().get("password").toCharArray());
+        }
+
+        sampleClient.connect(connOpts);
+        sampleClient.subscribe(topic, 1);
+        sampleClient.setCallback(new Receiver(clientId));
+        return sampleClient;
+    }
+
+    public void sendMessage(String messageContent, String topic) throws MqttException {
+        final String clientId = "syndesis-mqtt-sender";
+        MqttClient sampleClient = new MqttClient(broker, clientId, new MemoryPersistence());
+        MqttConnectOptions connOpts = new MqttConnectOptions();
+        connOpts.setCleanSession(false);
+
+        Optional<Account> optional = AccountsDirectory.getInstance().getAccount("QE MQTT");
+        if (optional.isPresent()) {
+            log.info("Setting username and password for QE MQTT client");
+            connOpts.setUserName(optional.get().getProperties().get("userName"));
+            connOpts.setPassword(optional.get().getProperties().get("password").toCharArray());
+        }
+
+        sampleClient.connect(connOpts);
+        MqttMessage message = new MqttMessage(messageContent.getBytes());
+        message.setQos(1);
+        sampleClient.publish(topic, message);
+        System.out.println("Message published from : " + clientId + " with payload of : " + messageContent);
+        sampleClient.disconnect();
+        sampleClient.close();
```

Comment:
It feels like we should handle exceptions and preferably use try-with-resources or at least finally block here.